src/components/screens/Home/NearByUserProfileScreen.js

```diff
@@ -0,0 +1,89 @@
+import {Image, StyleSheet, View, Text} from 'react-native';
+import {useTheme} from '@react-navigation/native';
+import React from 'react';
+import ProfilePicturesCarousel from './NearByUserProfilePicturesCarousel';
+import ApproachText from '../../ApproachText';
+import {ScrollView} from 'react-native-gesture-handler';
+import InterestsLabelsList from './InterestsLabelsList';
+import Location from '../../../../assets/icons/location.png';
+import ApproachButton from '../../ApproachButton';
+import Message from '../../../../assets/icons/dms.png';
+
+export function NearByUserScreen(props) {
```

Comment:
nit: can we do `export function NearByUserScreen({{ user }})` we have been deconstructing the props like this in other places, so best to keep it consistent.

Reply:
Also, on this same line, you might as well just 
```suggestion
export default function NearByUserScreen(props) {
```

Reply:
addressed